workflows/prognostic_c48_run/runtime/loop.py

```diff
@@ -187,10 +187,10 @@ def __init__(
         self.monitor = Monitor.from_variables(
             config.diagnostic_variables, state=self._state, timestep=self._timestep,
         )
-
-        self._emulate = runtime.factories.get_emulator_adapter(
-            config, self._state, self._timestep
+        self._adapt = runtime.factories.get_fv3_physics_adapter(
```

Comment:
to do both simultaneously:

```suggestion
        self._prescribe_tendency = runtime.factories.get_fv3_physics_adapter(
            ... args
        )
        self._emulate = runtime.factories.get_emulator_adapter(
            config, self._state, self._timestep)
        
```

Reply:
Sure, but we still need to decide which (or both or none) of these adapters is used to decorate `apply_physics`. The useful 'shared concept' between overriding and emulation in `loop.py` is that they are both being applied to the physics step.